pkg/cmd/pr/checks/checks.go

```diff
@@ -237,11 +227,11 @@ func populateStatusChecks(client *http.Client, repo ghrepo.Interface, pr *api.Pu
 		var resp response
 		err := apiClient.GraphQL(repo.RepoHost(), query, variables, &resp)
 		if err != nil {
-			return err
+			return nil, checkCounts{}, err
 		}
 
 		if len(resp.Node.StatusCheckRollup.Nodes) == 0 {
-			return nil
+			return aggregateChecks(pr, requiredChecks)
```

Comment:
Style tip: let's instead `break` here and allow the final `return` from the parent function to get reached.

Reply:
@mislav I quickly pushed last night without running tests. It turned out that `break` here did break tests. I reverted the commit for now.